src-web/components/kappnav/modals/ActionModal.js

```diff
@@ -116,11 +116,16 @@ const withForm = (Component, initialForm) => {
           document.getElementById("app").style.cursor = "pointer"; 
           if (response.ok) {
             if(response.status === 207) {
-              //Api Exception. One possible cause is that it failed to pull the image.
```

Comment:
We should keep this comment.  We no longer relate to the user that we failed to pull the image, but it doesn't hurt to keep that in the code.